suite/tests/linux/clone.c

```diff
@@ -47,7 +47,7 @@
 #include <string.h>
```

Comment:
> Dunno what the protocol is for specifying in the description the patch is the re-application of a reverted patch.
> Please advise.

There's nothing written down but it seems best to put a reference to the reverted and to the reverting PR numbers in the commit message so they'll all be linked.

Reply:
Done.